wp/plugin/wp.ml

```diff
@@ -218,7 +218,7 @@ let func_name_map = Cmd.parameter Typ.(list ~sep:';' (pair ~sep:',' string strin
            "<reg1_orig>,<reg1_mod>;<reg2_orig>,<reg2_mod>"). By default, WP
            assumes subroutines have the same names between the two binaries.|}
 
-let user_func_spec = Cmd.parameter Typ.(some (t3 string string string)) "user-func-spec"
+let user_func_spec = Cmd.parameter Typ.(list ~sep:';' (t3 string string string)) "user-func-spec"
     ~doc:{|Creates the weakest precondition for a subroutine given the
```

Comment:
Can you add how to write multiple user-defined specs and and an example?

Reply:
I also realized that we don't have that flag documented on our site. Can you add it here? https://github.com/draperlaboratory/cbat_tools/blob/master/docs/reference.html